misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsMetrics.kt

```diff
@@ -69,8 +69,22 @@ internal class SqsMetrics @Inject internal constructor(metrics: Metrics) {
   )
 
   val sqsDeleteTime = metrics.histogram(
-    "jobs_sqs_delete_latency",
-    "the round trip time to delete messages from SQS",
-    listOf("queueName", "QueueName")
+      "jobs_sqs_delete_latency",
+      "the round trip time to delete messages from SQS",
+      listOf("queueName", "QueueName")
+  )
+
+  val sqsApproxNumberOfMessages = metrics.gauge(
```

Comment:
I think ideally these should match the metrics that we consume directly from Cloudwatch into signalfx, so the same dashboards can be used for external queues.

Reply:
updated